assets/js/hooks/cell_editor.js

```diff
@@ -38,6 +42,17 @@ const CellEditor = {
           const skeletonEl =
             editorContainer.querySelector(`[data-el-skeleton]`);
           skeletonEl && skeletonEl.remove();
+
+          if (settings.editor_vim_mode) {
+            this.vimMode = initVimMode(this.liveEditor.editor);
+            this.vimMode.on("vim-mode-change", ({"mode": mode}) => {
+              editorEl.setAttribute("data-vim-mode", mode);
+            });
+          } else if (settings.editor_emacs_mode) {
+            this.emacsMode = new EmacsExtension(this.liveEditor.editor);
+            this.emacsMode.start();
+            unregisterKey("Tab");
+          }
```

Comment:
Let's move this to `_mountEditor` in `LiveEditor`. We can set the attribute on the editor container there :)